packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -1632,6 +1642,8 @@ export class InstanceType {
       [InstanceClass.MAC2_M2]: 'mac2-m2',
       [InstanceClass.MACINTOSH2_M2_PRO]: 'mac2-m2pro',
       [InstanceClass.MAC2_M2PRO]: 'mac2-m2pro',
+      [InstanceClass.MACINTOSH2_M1_ULTRA]: 'mac2-m1ultra',
```

Comment:
should the value be `macintosh2-m1-ultra` ?

Reply:
I've fixed!